softdevice_controller/include/ble_controller_soc.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#ifndef BLE_CONTROLLER_SOC_H__
+#define BLE_CONTROLLER_SOC_H__
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include "sdc_soc.h"
+
+#define BLE_CONTROLLER_FLASH_CMD_STATUS         SDC_SOC_FLASH_CMD_STATUS
```

Comment:
should SDC_SOC_FLASH_CMD_STATUS be lower-case?

Enums usually are lower-case named in both NCS and Zephyr,, SoftDevice controller uses upper case.